### PR TITLE
Update bacon bdwgc byobu libatomic_ops

### PR DESCRIPTION
**Pull request by rmrfslashbin**
* bacon -> 3.7.1
* bdwgc -> 7.6.4
* byobu -> 5.124
* libatomic_ops -> 7.6.2

## Description
Updated packages as described.

### CODE REVIEW

File: packages/bdwgc.rb [hunk 1]
```diff
@@ -3,26 +3,19 @@
...
-    aarch64: '4efb53da6a7755b5f29badf95618bc40642505ae04fab659d7f63dab533b79af',
-     armv7l: '4efb53da6a7755b5f29badf95618bc40642505ae04fab659d7f63dab533b79af',
-       i686: '0b99de0c290eb01e859ac762146266966e92fc39a07ed2968490fc8796125953',
-     x86_64: 'a3df6a1250d94d92b3974c35b7d709e5362acec7379800ca3f09d3b0e492c2e3',
   })
 
   depends_on 'libatomic_ops'
 
   def self.build
+      system "./autogen.sh"
```
Comment (uberhacker) [inline packages/bdwgc.rb]: If you add this, I'm pretty sure you also need autoconf, automake and maybe gettext.  Can you please confirm and add these as build dependencies if necessary?  What was your issue without this line?

File: packages/libatomic_ops.rb [hunk 2]
```diff
@@ -3,24 +3,17 @@
...
   })
   binary_sha256 ({
-    aarch64: 'ccbc4989f3ac6368de4f7c50dc419ec91c00cdcd3038fd42aa7bc05d93f666af',
-     armv7l: 'ccbc4989f3ac6368de4f7c50dc419ec91c00cdcd3038fd42aa7bc05d93f666af',
-       i686: '975680fbbd2aa398ee2cfeada7251e22c80595f8cf4a19b3b0fc9c43bcc07bd0',
-     x86_64: 'ec3cc2c64f4a730b931ca414a4d66f0789ff951c4d6f7e4bc549f0ca6ac72688',
   })
 
   def self.build
+    system "./autogen.sh"
```
Comment (uberhacker) [inline packages/libatomic_ops.rb]: Please see comment in the bdwgc package.

File: packages/libatomic_ops.rb [hunk 3]
```diff
@@ -3,24 +3,17 @@
...
     system "./configure"
```
Comment (uberhacker) [inline packages/libatomic_ops.rb]: Can you please add `--prefix=#{CREW_PREFIX}` and `--libdir=#{CREW_LIB_PREFIX}` if there are any library files?

File: packages/bacon.rb [hunk 4]
```diff
@@ -3,21 +3,13 @@
...
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bacon-3.6-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '468b7c67d22370fcd6fc5ff6b675a07686a74a1c7e5948e7c787f6e22ff78761',
-     armv7l: '468b7c67d22370fcd6fc5ff6b675a07686a74a1c7e5948e7c787f6e22ff78761',
-       i686: 'ef5f034044d91aeb9954f48a0687840850d661bfe57821504ff41a242ea46ed3',
-     x86_64: '504374ac5978d4d203c06bdce9436cd3ef6839d7e4c441ef58b3440b7e7fd982',
   })
 
   depends_on 'buildessential'
```
Comment (uberhacker) [inline packages/bacon.rb]: This line can be removed.

**Comment by uberhacker** [comment]
@rmrfslashbin: Please see the inline comments in the `Files changed` section.

File: packages/bdwgc.rb (hunk 1 above)
Comment (rmrfslashbin) [reply packages/bdwgc.rb]: @uberhacker Yes, a run of ```autogen.sh``` is required as the tarball does not contain a ready-made ```configure```. I added the build deps, as you noted.

File: packages/libatomic_ops.rb (hunk 2 above)
Comment (rmrfslashbin) [reply packages/libatomic_ops.rb]: @uberhacker same issue as the bdwgc package; tarball missing ```configure```.

File: packages/libatomic_ops.rb (hunk 3 above)
Comment (rmrfslashbin) [reply packages/libatomic_ops.rb]: @uberhacker good catch, added.

**Pull request merged**